### PR TITLE
RE2022-208: include sample id

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

**Review by MrCreosote — COMMENTED** [review]
LGTM assuming GHA passes

**Comment by Tianhao-Gu** [comment]
@MrCreosote Still need an approval stamp to unblock the merge.

**Review by MrCreosote — APPROVED** [review]
Whoops, I guess I forgot to toggle to approve

**Pull request merged**
